cmd/operator-sdk/internal/genutil/k8s.go

```diff
@@ -34,6 +35,16 @@ import (
 func K8sCodegen() error {
 	projutil.MustInProjectRoot()
 
+	out, err := exec.Command("go", "env", "GOROOT").CombinedOutput()
```

Comment:
nit:  (to make more clear what it is)
```suggestion
	goEnv, err := exec.Command("go", "env", "GOROOT").CombinedOutput()
```

Reply:
Done, good suggestion!